src/test/mempool_tests.cpp

```diff
@@ -566,11 +566,19 @@ BOOST_AUTO_TEST_CASE(MempoolSizeLimitTest)
     BOOST_CHECK_EQUAL(pool.GetMinFee(pool.DynamicMemoryUsage() * 9 / 2).GetFeePerK(), (maxFeeRateRemoved.GetFeePerK() + 1000)/8);
     // ... with a 1/4 halflife when mempool is < 1/4 its target size
 
-    SetMockTime(42 + 7*CTxMemPool::ROLLING_FEE_HALFLIFE + CTxMemPool::ROLLING_FEE_HALFLIFE/2 + CTxMemPool::ROLLING_FEE_HALFLIFE/4);
+    unsigned i = 5;
+    while (pool.GetMinFee(1).GetFeePerK() > (int)(2 * KB)) {
```

Comment:
It wa clearer when it was CFeeRate::INTERNAL_PRECISION than now that it's KB...
I can add comments, suggested comments always preferred, otherwise I'll come up with something myself ...There's one comment about the internal prevision below, but maybe that's not clear enough.